core-plugin/build.gradle

```diff
@@ -46,7 +46,10 @@ sourceSets.test.java.srcDirs = ['testSrc']
 
 test {
     testLogging {
-        events "passed", "skipped", "failed", "standardOut", "standardError"
+        events "skipped", "failed"
+    }
+    doLast{
```

Comment:
I'm not sure a doLast block will get executed if tests are failing. http://stackoverflow.com/questions/20973365/gradle-execute-task-after-test-phase-even-if-test-has-failed

On the other hand, the test result link is printed to screen on failure by the system, so maybe it doesn't matter?